File: examples/comparison/nested/Nested.java
public class Nested {
    public static void main(String[] args) {
        long sum = 0;
        final int n = 4000;
        final long mod = 1_000_000_007L;
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                sum = (sum + (long) i * j) % mod;
            }
        }
        System.out.println(sum);
    }
}

File: examples/comparison/nested/nested.js
const n = 4000;
const MOD = 1_000_000_007;
let sum = 0;
for (let i = 0; i < n; i++) {
  for (let j = 0; j < n; j++) {
    sum = (sum + i * j) % MOD;
  }
}
console.log(sum);
